infra/examples-dev/aws-msft-365/main.tf

```diff
@@ -168,8 +139,10 @@ module "psoxy-msft-connector" {
   aws_account_id        = var.aws_account_id
   path_to_repo_root     = var.psoxy_base_dir
   environment_variables = {
-    PSEUDONYMIZE_APP_IDS = tostring(var.pseudonymize_app_ids)
     IS_DEVELOPMENT_MODE  = "true"
+    CLIENT_ID            = module.msft-connection[each.key].connector.application_id
+    REFRESH_ENDPOINT     = "https://login.microsoftonline.com/${var.msft_tenant_id}/oauth2/v2.0/token"
```

Comment:
Probably this URL could be part of connector specs module output for MSFT instead of being hardcoded here and in the other example